### PR TITLE
Hossein auto complete scrolling bug #460

### PR DESCRIPTION
**Pull request by hoseinmirian**

### CODE REVIEW

**Review by knirirr — COMMENTED** [review]
Is this commit to fix the problem of autocomplete dropdowns staying in position when the screen scrolls? If so then I'm still seeing the bug on my system when I check this branch out.

**Comment by terazus** [comment]
> 
> 
> Is this commit to fix the problem of autocomplete dropdowns staying in position when the screen scrolls? If so then I'm still seeing the bug on my system when I check this branch out.

same here

**Comment by hoseinmirian** [comment]
> 
> 
> Is this commit to fix the problem of autocomplete dropdowns staying in position when the screen scrolls? If so then I'm still seeing the bug on my system when I check this branch out.

yes it is. it should be fine.

**Review by terazus — CHANGES_REQUESTED** [review]
Not working as intended, the behaviour is still here (only the scroll bar has disaper. You now need to scroll locking of the left panel.
After that you to also sort the two position: small height and full height.

**Comment by knirirr** [comment]
The latest change does appear to fix the scroll issue, which is good. However, it has affected the appearance of elements within the dropdown, which now seem to overflow (I'm using Firefox, BTW). Counts with one digit fit, two causes the element to touch the edge of the dropdown, and a third digit causes the overflow.

Screenshot 2020-10-02 at 13 45 41
Screenshot 2020-10-02 at 13 41 57

**Review by knirirr — APPROVED** [review]
Please wait for @terazus to review also before merging - thanks.

**Review by terazus — CHANGES_REQUESTED** [review]
Please merge from dev before i do the review :)

File: src/components/Records/Search/Input/SearchInput.vue
Comment (terazus) [inline src/components/Records/Search/Input/SearchInput.vue]: Please keep these commented functions either on a local branch or on a separate branch. What you submit ends up on master and unused code shouldn't be merged.

File: src/components/Records/Search/Input/FilterAutocomplete.vue
Comment (terazus) [inline src/components/Records/Search/Input/FilterAutocomplete.vue]: Stroustrup on all if/else catch/try plz.

File: tests/unit/store/uiController.spec.js
Comment (terazus) [inline tests/unit/store/uiController.spec.js]: use haveBeenCalledWith **if possible** for all these tests.

**Review by terazus — CHANGES_REQUESTED** [review]
Minor modifications and we are good to merge.

**Review by terazus — APPROVED** [review]

**Pull request merged**
